### ISSUE TITLE
Upgrade to Spring Cloud Greenwich.RELEASE

### ISSUE THREAD

**Issue by csterwa** [pivotal-cf/spring-cloud-services-connector#91]
https://spring.io/blog/2019/01/23/spring-cloud-greenwich-release-is-now-available

**Comment by habuma** [comment]
Should this also include an upgrade to Boot 2.1.2? (Connectors are currently on Boot 2.1.1. Boot 2.1.2 was released on Jan 12th.)

**Comment by csterwa** [comment]
Please upgrade to most recent.

**Comment by habuma** [comment]
It seems that Greenwich.RELEASE, by way of org.springframework.cloud:spring-cloud-security-dependencies:2.1.0.RELEASE defines a dependency management against a milestone release of spring-security-oauth2-autoconfigure. I raised the issue in the #spring-cloud Slack channel and they consider it a bug. I'll be submitting an issue, but it's uncertain when a SR will be available.

Is this a "stop-the-presses" event for SCS connectors 2.1.0.RELEASE or should we move forward as planned?

**Comment by habuma** [comment]
Issue created: https://github.com/spring-cloud/spring-cloud-security/issues/172

Spencer says that they can likely cut a quick 2.1.1.RELEASE of spring-cloud-security-dependencies. We can then build our connectors/starters with that explicit version with no need to wait on Greenwich.SR1.

**Comment by sotcsa** [comment]
How to use this fixed version 2.1.1.RELEASE with maven? Currently I got the same issue...
```
      <dependency>
        <groupId>org.springframework.cloud</groupId>
        <artifactId>spring-cloud-dependencies</artifactId>
        <version>Greenwich.RELEASE</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```
Many thanks.

**Comment by csterwa** [comment]
@sotcsa In order to use `Greenwich.RELEASE` you must use Spring Boot `2.1` and SCS Connectors version `2.1.1`. These would replace versions that we show in the current SCS docs at https://docs.pivotal.io/spring-cloud-services/2-0/common/client-dependencies.html#including-dependencies. Please let me know if you have any follow up questions or issues with this.
